app/controllers/UTRController.scala

```diff
@@ -105,17 +103,20 @@ class UTRController @Inject() (
               .bindFromRequest()
               .fold(
                 formWithErrors => render(mode, regime, formWithErrors, businessType).map(BadRequest(_)),
-                value =>
+                value => {
+                  val previousAnswers = request.userAnswers.get(UTRPage).contains(value)
```

Comment:
@iservport 
```suggestion
                  val previousAnswers = request.userAnswers.get(UTRPage).contains(value)
```
Please remove unused val previousAnswers

Reply:
Removed